pages/training/modules/index.js

```diff
@@ -0,0 +1,41 @@
+const { get, intersection } = require('lodash');
+const { page } = require('@asl/service/ui');
+const { form } = require('../../common/routers');
+const { submit } = require('../middleware');
+const getSchema = require('./schema');
+
+const SPECIES_MODULES = [
+  'PILA (theory)',
+  'PILA (skills)',
+  'K (skills)'
+];
+
+module.exports = () => {
+  const app = page({ root: __dirname });
+
+  app.use((req, res, next) => {
+    form({
+      configure(req, res, next) {
+        const isExemption = get(req.session, `form[${req.model.id}].values.isExemption`);
+        req.form.schema = getSchema(isExemption);
+        next();
+      },
+      locals(req, res, next) {
+        res.locals.static.isExemption = get(req.session, `form[${req.model.id}].values.isExemption`);
+        next();
+      }
+    })(req, res, next);
+  });
```

Comment:
This is nested inside itself in a weird way. Just do `app.use(form({...}))` directly.